get-sysinternals-du.js

```diff
@@ -5,45 +5,82 @@ const path = require('path')
 const decompress = require('decompress')
 const { HttpsProxyAgent } = require('https-proxy-agent')
 
-// Only run for Windows
-if (process.platform !== 'win32') {
-  process.exit(0)
-}
+exports.workspace = __dirname
```

Comment:
I would prefer a different approach for this. In general, avoid mutable state if you can. Instead of having this variable as an export, accept it as an argument to the default export, defaulting to __dirname. this will allow testing without using mutable state

Reply:
Done.